dat/generated_tables.py

```diff
@@ -35,7 +40,7 @@ def save_expected(case: TestCaseInfo, as_latest=False) -> None:
     # Need to ensure directory exists first
     os.makedirs(case.expected_root(version))
 
-    df.toPandas().to_parquet(case.expected_path(version))
```

Comment:
It turned out `toPandas().to_parquet()` causes weird things to happen to timestamps, so better to stick to Spark here. Without this, I was able to eliminate the pandas dependency as well.